srcpkgs/gocryptfs/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'gocryptfs'
+pkgname=gocryptfs
+version=1.6
+revision=1
+wrksrc="${pkgname}_v${version}_src-deps"
+build_style=go
+go_import_path="github.com/rfjakob/gocryptfs"
+go_build_tags=without_openssl
```

Comment:
Quote this. Sorry I was not clear in my comment earlier.

Reply:
sorry, I should have seen other examples first, hopefully it's fixed now